client/src/Component/SignUp/dataToServer.js

```diff
@@ -0,0 +1,15 @@
+const dataToServer = async (formData) => {
```

Comment:
how are you handling errors here? like network error or anything?

Reply:
The error sent from server as a customize response, is shown to the user. We will make it more customize and informative to the user once we are clear with all error cases.